SPECS/iscsi-initiator-utils/iscsi-initiator-utils.spec

```diff
@@ -1,25 +1,26 @@
 %global __provides_exclude_from ^(%{python2_sitearch}/.*\\.so|%{python3_sitearch}/.*\\.so)$
+%global debug_package %{nil}
```

Comment:
Why are we removing the debug package ?

Reply:
it was leftover from mock debugging, due to an issue with mock (bug 38136818)